### ISSUE TITLE
Feedback colorful problem

### ISSUE THREAD

**Issue by tcqq** [garretyoder/Colorful#22]
1. How to get primary and accent color res?
1. ThemeColor have't white and black color `ThemeColor.WHITE`,  how to set primary or accent color to white or black color?

**Comment by stcojo** [comment]
Is this your first day coding for Android?

**Comment by tcqq** [comment]
@cojoc07 These methods exist in the previous version, Do you have any other solution to the above problem?

**Comment by garretyoder** [comment]
```kotlin
Colorful().getPrimaryColor().getColorPack().light().asInt()
```

I recommend you use the power of your IDE's code completion to poke around Colorful and Android's platform APIs.

**Comment by tcqq** [comment]
@garretyoder Thanks, now I know should be how to get primary and accent color:-), But I didn't find white or black in `ThemeColor`, How should I set PrimaryColor to white or black?

**Comment by garretyoder** [comment]
White or black is something I didn't add in v2 because I didn't see a point. If it's something you really require, I can add it this weekend for v2.1 because it'll only take me a few moments.

**Comment by tcqq** [comment]
Yes, white and black are useful, see the picture below.

`Primary Color: BLUE`
`Accent Color: PINK`
`Theme: DARK`
![image](https://user-images.githubusercontent.com/10644057/38438013-cb153dc6-3a0b-11e8-9d40-8c1134ac4c5f.png)

`Primary Color: WHIT`
`Accent Color: BLUE`
`Theme: LIGHT`
![image](https://user-images.githubusercontent.com/10644057/38438160-4ca35896-3a0c-11e8-9c5f-b9536d68e860.png)

**Comment by garretyoder** [comment]
White/Black are now added courtesy of a PR, which will be included in the 2.1 release that I'll probably push later today.

**Comment by tcqq** [comment]
Thanks:)
